charts/secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml

```diff
@@ -46,7 +46,6 @@ spec:
         - name: secrets-store
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           args:
-            - "--v=3"
```

Comment:
Should we default with `--debug=true` here? also the same change needs to be updated in the yaml in `/deploy`

Reply:
A lot of debug logs we had were set at v=5 log level. So I think by default we should leave `debug=false` and enable only if required. 

We should update in `/deploy` too but only when we have a new image for `secrets-store-csi`. B

Reply:
sgtm